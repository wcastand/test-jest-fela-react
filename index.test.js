import React from "react";
import createSerializer from "jest-fela-react";
import { createRenderer } from "fela";
import { Provider, createComponent } from "react-fela";
import monolithic from "fela-monolithic";
import renderer from "react-test-renderer";

const felaRenderer = createRenderer();

expect.addSnapshotSerializer(createSerializer(felaRenderer));

test("test 1", () => {
  const Test = createComponent(() => ({ fontSize: '16px', color: "red" }));
  const tree = renderer
    .create(<Provider renderer={felaRenderer}>
        <Test>Test</Test>
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
