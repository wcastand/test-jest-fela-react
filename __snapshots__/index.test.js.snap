// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1 1`] = `
.a {
  font-size: 16px;
}

.b {
  color: red;
}

<div
  className="a b"
>
  Test
</div>
`;
